### ISSUE TITLE
BF: When exporting HTML, Slider's style property should be exported as Array, not String (PsychoJS expects Array)

### ISSUE THREAD

**Issue by sijiazhao** [psychopy/psychopy#3636]
Versions tested: 2021.1.1 (bug), 2020.2.10 (no bug)

PsychoJS expects Slider's style property to be given as an `Array`, however PsychoPy exports the style property as a `String`.

Therefore causes unrecoverable `TypeError`s whenever `forEach()` is called on the `._style` property (`forEach()` only works with Arrays).

I believe this can be fixed in PsychoPy by modifying [`psychopy/experiment/components/slider/__init__.py` Line 265](https://github.com/psychopy/psychopy/blob/585e2e8d6d4e633c5d6b19bee9f3528e899a89c1/psychopy/experiment/components/slider/__init__.py#L265):

```
                    "  granularity: {granularity}, style: {styles},\n"
```

to:
```
                   "  granularity: {granularity}, style: [{styles}],\n"
```

It seems to work in my quick testing. Other option would be to change PsychoJS to expect a `String` or check `Array.isArray(this._style)` every time if both types are to be accepted.

I haven't made a PR for this because my expertise is JavaScript, but I'd be happy to submit one if you find my proposal reasonable.

**Comment by peircej** [comment]
Thanks, looking into it now. Yes, @TEParsons has been moving the Slider to have a single style and then potential 'tweaks' on top because it wasn't clear which of the existing combinations were/weren't mutually exclusive. Unfortunately I think that work has moved into the release before the JS had been updated to match

**Comment by peircej** [comment]
thanks. They should also be in cap and shoudl include the new "tweaks" (separating the mutually exclusive and the additive styles settings).

All fixed in #3644 hopefully!
